README.md

```diff
@@ -366,6 +365,48 @@ enable DEBUG logging in the SDK. You can add your own log statements in
 your workflow for further identification of how your workflow script
 is progressing.
 
+## TLS Capability
+
+The websocket server that is built-in to this Python Relay SDK has support
+for providing TLS encryption, given that you have a TLS key and certificate.
+For example, you can get a TLS key and certificate from letsencrypt.org. Or
+you can use a commercial provider. Sometimes this key and certificate for
+HTTP-over-TLS is called an "SSL certificate".
+
+To use the built-in TLS support, the key and certificate both need to
+be in PEM format, and in a file that can be read locally by the workflow
+process.
+
+When not using TLS support, your workflow app should have a line that
+looks like this:
+
+`server = relay.workflow.Server('0.0.0.0', port)`
+
+When you want to use the built-in TLS support, you supply extra arguments
+to this line that identify the filename of both the key and the
+certificate:
+
+    my_ssl_key_filename = '/etc/letsencrypt/live/myhost.mydomain.com/privkey.pem'
+    my_ssl_cert_filename = '/etc/letsencrypt/live/myhost.mydomain.com/fullchain.pem'
+    server = relay.workflow.Server('0.0.0.0', port, ssl_key_filename=my_ssl_key_filename, ssl_cert_filename=my_ssl_cert_filename)
+
+Now when you start your workflow application, in the log you should see this line:
+
+`Relay workflow server (relay-sdk-python/2.0.0) listening on 0.0.0.0 port 8080 with ssl_context MySslContext`
+
+When it says "ssl_context" instead of "plaintext", that is the clue that TLS is enabled.
+
+If you want to verify the certificate that your server is presenting to web clients, run:
+
+`openssl s_client -debug myserver.mydomain.com:8080`
+
+...assuming that you are still specifying port 8080. You may consider changing the port in
```

Comment:
might be easier for the reader if we just present this with port == 443.
changing it seems like an advanced use case -- leave that to the adventurous, but no need to muddy this section?

Reply:
Does the 2nd commit address your point? I do agree it is an improvement.